Lib/Slatec/slatec.pd

```diff
@@ -1270,12 +1270,12 @@ defslatec(
           ',
 'Evaluate the definite integral of a a piecewise
 cubic Hermite function over an arbitrary interval,
-given by C<[$a,$b]>.
+given by C<[$x,$y]>.
```

Comment:
I am concerned about these changes given the presence of `x` in the defslatec call.  Have you had a chance to run this or look at the generated code?  If not, I can.  Would t/slatec.t pass tests?  Not sure if Travis builds slatec as a part of its PDL build.

Reply:
Just fixed the chia and chid documentation.